src/actions/comments.js

```diff
@@ -43,12 +44,23 @@ export function appendComment (postId, comment) {
   }
 }
 
-export function createComment ({ postId, text, tagDescriptions, imageUrl }) {
+export function createComment ({ postId, text, tagDescriptions, imageUrl, userId }) {
   const params = {text, tagDescriptions, imageUrl}
+  const comment = {
```

Comment:
yeah, that's what I had it as initially, but then for some reason when I was talking with Robbie I decided to name it this. I think looking back now I agree with you though, so I'll do that